src/Ormolu/Fixity/Internal.hs

```diff
@@ -79,8 +95,36 @@ instance Semigroup FixityInfo where
           (Just a, Just b) | a == b -> Just a
           _ -> Nothing
 
+-- | An operator name.
+newtype OpName = MkOpName
+  { -- | Invariant: UTF-8 encoded
+    getOpName :: ShortByteString
```

Comment:
Is there a difference in performance if this is `Text` instead? Why should it be a `ShortByteString` (or even `ShortText`)?

Reply:
I would be surprised if there were a big difference in all cases other than synthetic benchmarks. My motivation to use `ShortByteString` was that GHC already uses them internally, so converting them to `OpName` is free, see `occOpName`.